code/index.html

```diff
@@ -1,22 +1,228 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+
+
+  <header>
+
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <div class="overlay-content">
+        <a href="#">News</a>
+        <a href="#">World</a>
+        <a href="#">Local</a>
+        <a href="#">Cultur</a>
+        <a href="#">Politics</a>
+        <a href="#">Sport</a>
+        <a href="#">Finance</a>
+        <a href="#">Health</a>
+        <a href="#">Tech</a>
+        <a href="#">ABOUT</a>
+      </div>
+    </div>
+
+    <span style="font-size:40px;cursor:pointer" onclick="openNav()">&#9776; </span>
+    <script>
+      function openNav() {
+        document.getElementById("myNav").style.width = "100%";
+      }
+      function closeNav() {
+        document.getElementById("myNav").style.width = "0%";
+      }
+    </script>
+
+    <h1>NEWS TIME</h1>
+
+    </div>
+    </div>
+  </header>
+
+  <!--THE NAV MENU UNDER THE HEADER FROM 668px and up-->
+  <nav class="menu">
+    <a href="#">News</a>
+    <a href="#">World</a>
+    <a href="#">Local</a>
+    <a href="#">Cultur</a>
+    <a href="#¢">Politics</a>
+    <a href="#">Sport</a>
+  </nav>
+
+
+
+
+  <section class="cards">
+
+    <div class="article-one">
+        <div class="card">
+          <img src="pexels-pixabay-73821.jpg" alt="Hexbot image">
+          <div class="fadedbox">
+            <div class="title text"><button>Article</button> 
+            </div>
+          </div>
+        </div>
+    </div>
+    </a>
+
+    <div class="article-two">
+        <div class="card">
+          
+          <h4>Big Article</h4>
+          <h5>Nearly 50 dead after Afghan downpour.At least 46 people have been killed, about 80 have been injured in the city of Charikar north of
```

Comment:
Try to use your heading in order, you already have a h1 so this should probably be a h2 and h3. It's good practise!